src/examples/src/widgets/radio/Labelled.tsx

```diff
@@ -4,5 +4,11 @@ import Radio from '@dojo/widgets/radio';
 const factory = create();
 
 export default factory(function LabelledRadioButton() {
-	return <Radio label={'Radio Button 1'} />;
+	return (
+		<Radio>
```

Comment:
I'd expect this to be `<Radio>Radio Button 1</Radio>`

Reply:
Updated this per feedback. Simplified implementation to eliminate render function, etc.